specification/iotoperations/resource-manager/readme.md

```diff
@@ -47,16 +47,18 @@ directive:
     reason: User defined properties that are not subject to any validations.
   - suppress: BodyTopLevelProperties
     reason: Temporary suppression due to failing pipeline.
+  - suppress: PatchBodyParametersSchema
```

Comment:
>PatchBodyParametersSchema [](http://example.com/codeflow?start=14&length=25)

Can you specify `where` property here instead of suppressing globally?